mw-webapp/src/component/textarea/Textarea.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import {useState} from "react";
+import {Textarea} from "src/component/textarea/Textarea";
+
+const meta = {
+  title: "Textarea",
+  component: Textarea,
+  parameters: {layout: "centered"},
+  tags: ["autodocs"],
+};
+
+export default meta;
+
+/**
+ * Default story for the Textarea component.
+ */
+export const Default = () => {
+  const [text, setText] = useState<string>("");
```

Comment:
get rid of value (use default value instead)

Reply:
Done